cmake/wil.cmake

```diff
@@ -0,0 +1,2 @@
+add_library(wil INTERFACE)
```

Comment:
Please add license header and copyright notice.

Reply:
done